canonical/specs-v1.canonical.com
# Revert "Improve specs list UI performance (#184)"

This reverts commit 091cb7274c0b1f40cc9059ef240d3902baf151ad.

The last PR works fine except for the side panel (spec details), that looks broken (and I didn't realise when QAing). Reverting until we find a solution.

## QA

Go to https://specs.canonical.com/, click on one spec, check that the side panel doesn't work properly.
